Scripts/SL-Helpers.lua

```diff
@@ -572,3 +572,83 @@ end
 IsAutoplay = function(player)
 	return GAMESTATE:GetPlayerState(player):GetPlayerController() ~= "PlayerController_Human"
 end
+
+
+-- -----------------------------------------------------------------------
+CalculateExScore = function(player)
+	local pn = ToEnumShortString(player)
+	local stats = STATSMAN:GetCurStageStats():GetPlayerStageStats(pn)
+
+	local total_points = 0
+	local total_possible = 0
+
+	local TNS = { "W1", "W2", "W3", "W4", "W5", "Miss" }
+
+	-- First get tap notes.
+	for index, window in ipairs(TNS) do
+		local number = stats:GetTapNoteScores( "TapNoteScore_"..window )
+
+		-- The W0 window needs to be emulated in ITG mode.
+		-- We already track note offsets so use that to compute the W0 count.
+		-- TODO(teejusb): Explicitly track this value instead of computing the count here
+		-- as it will be useful to display the count during gameplay.
+		if window == "W1" and SL.Global.GameMode == "ITG" then
+			local offsets = SL[ToEnumShortString(player)].Stages.Stats[SL.Global.Stages.PlayedThisGame + 1].sequential_offsets
```

Comment:
`SL[pn]`

Reply:
Using sequential_offsets will not work in casual mode, because judgment offsets are not being tracked.
https://github.com/Simply-Love/Simply-Love-SM5/blob/release/BGAnimations/ScreenGameplay%20overlay/JudgmentOffsetTracking.lua#L9

Reply:
Done, and also handled casual mode separately (I just return early, but also it should technically not even be called now).